ram/wram.asm

```diff
@@ -1804,6 +1804,12 @@ wMapDataPtr:: dw
 wMapTextPtr:: dw
 wMapScriptPtr:: dw
 
+	const_def
```

Comment:
Constants should be defined in constants/\*_constants.asm files. See wram_constants.asm in pokegold and pokecrystal.